src/sign-in-xplat/MainPage.xaml.cs

```diff
@@ -23,6 +24,8 @@ public partial class MainPage : ContentPage
 
         private static readonly string s_graphURL = "https://graph.microsoft.com/v1.0/";
         private static AuthenticationResult s_authResult;

```

Comment:
This doesn't seem to be used at a global level anymore.  Is there a reason it's still out here?  Can `s_authResult` just be inlined into the function below?

Reply:
good catch @ckittel we can inline this as well

done | addressed from [70276](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/17/commits/70276147f1bced1913058358abc4706902e7cb50)